test/parallel/test-fs-utimes.js

```diff
@@ -162,3 +169,22 @@ process.on('exit', function() {
   console.log('Tests run / ok:', tests_run, '/', tests_ok);
   assert.strictEqual(tests_ok, tests_run);
 });
+
+if (common.isWindows) {
+  // Ref: https://github.com/nodejs/node/pull/12607
+  for (const date of [
+                      new Date('2017-04-08T17:59:38.018Z'),
+                      new Date('2017-04-08T17:59:38.008Z')]) {
+    const filename = path.join(common.tmpDir, `file-${Number(date)}`);
```

Comment:
Because this test uses `common.tmpDir`, it needs to call `common.refreshTmpDir()`.